app/locales/en/translations.js

```diff
@@ -57,6 +59,7 @@ export default {
         see_less: 'See less',
         version: 'Version',
         preprint_doi: '{{documentType.singularCapitalized}} DOI',
+        preprint_doi_default: 'Preprint DOI',
```

Comment:
Just wondering, but wouldn't this be the same as `preprint_doi`?  Where is this being used?

Reply:
`preprint_doi` uses `documentType` to determine the correct preprint word. `documentType` is an attribute of a provider. However, there is no provider related information for withdrawn preprints, meaning that we don't really know what the preprint word is, in which case it should fall back to using `preprint_doi_default`, which uses the default preprint word.

Reply:
Was that confirmed with product somewhere? I'm wondering if just "DOI" is sufficient. Alternatively, we do use "Paper" as a generic word in a few places now, so it could be "Paper DOI". Confirm with them and update as appropriate. It is a bit weird if, for example, a withdrawn Thesis said "Preprint DOI".

Reply:
"Preprint DOI" is used in the mockup. This went through demo and product did not raise any concern about the wording. I'll however check with product again to make sure.